connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -193,6 +193,7 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper {
   }
 
   test("write jdbc") {
+    assume(IntegrationTestUtils.isSparkHiveJarAvailable)
```

Comment:
`derby.jar` will only be copied to `assembly/target/scala-xxx/jars/` when build with -Phive, so there check `isSparkHiveJarAvailable`, maybe a little trick.
  
Another way is to check if  `derby-xxx.jar` is in the `assembly/target/scala-xxx/jars/` directory.